Index: OpenGL-basico/entities/player.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#pragma once\r\n#include \"entity.h\"\r\n\r\nclass player : public entity\r\n{\r\npublic:\r\n    explicit player(): entity(\"../assets/models/Doodler.obj\", \"../assets/textures/Doodler.png\")\r\n    {\r\n    }\r\n};\r\n
===================================================================
diff --git a/OpenGL-basico/entities/player.h b/OpenGL-basico/entities/player.h
--- a/OpenGL-basico/entities/player.h	
+++ b/OpenGL-basico/entities/player.h	
@@ -6,5 +6,6 @@
 public:
     explicit player(): entity("../assets/models/Doodler.obj", "../assets/textures/Doodler.png")
     {
+        
     }
 };
